inc/saiacl.h

```diff
@@ -339,8 +437,12 @@ typedef enum _sai_acl_table_attr_t
     /** Range type defined in sai_acl_range_type_t */
     SAI_ACL_TABLE_ATTR_FIELD_RANGE,
 
+    /** List of actions in sai_acl_table_action_list_t [sai_u32_list_t] 
+     * Default action list will be present which can be modified*/
+    SAI_ACL_TABLE_ATTR_ACTION_LIST,
```

Comment:
When you say the default action list, will it be defined by SAI proposal or the provider of the adapter will init this to what he can support. Will a adapter host select a subset from this. I thing just being able to query the action list (read only parameter) should be sufficient for the application to adapt its logic.